static/lang/re-en.json

```diff
@@ -2099,6 +2099,9 @@
                 "Forager": {
                     "Note": "While using Survival to Subsist, if you roll any result worse than a success, you get a success. You can also provide for multiple creatures."
                 },
+                "KnowItAllEldritchResearcher": {
+                    "Note": "When you succeed at a check to Recall Knowledge, you gain additional information or context. When you critically succeed at a Knowledge check, at the GM's discretion you might gain even more additional information or context than normal."
+                },
```

Comment:
We have a section for degree of success adjustments already, let's put this in there as a more generic entry.

Reply:
I moved it to the requested section and made it a bit more generic.